packages/webhooks/schema/models/WebhookPayload.js

```diff
@@ -0,0 +1,168 @@
+            access: {
+                create: true,
+                read: true,
+                update: false,
+            },
+        },
+
+        eventType: {
+            schemaDoc: 'Type of event that triggered this webhook. Use dot-notation format: "{resource}.{action}" or "{resource}.{sub-resource}.{action}". Examples: "payment.created", "payment.status.changed", "invoice.paid", "user.deleted". Use lowercase with dots as separators.',
+            type: 'Text',
+            isRequired: true,
+            hooks: {
+                validateInput: ({ resolvedData, fieldPath, context }) => {
+                    const value = resolvedData[fieldPath]
+                    if (value) {
+                        const { isValid, error } = validateEventType(value)
+                        if (!isValid) {
+                            throw new GQLError({ ...ERRORS.INVALID_EVENT_TYPE, message: error }, context)
+                        }
+                    }
+                },
+            },
+        },
+
+        modelName: {
+            schemaDoc: 'Name of the model that triggered this webhook (e.g., "Payment")',
+            type: 'Text',
+            isRequired: false,
+        },
+
+        itemId: {
+            schemaDoc: 'ID of the record that triggered this webhook',
+            type: 'Uuid',
+            isRequired: false,
+        },
+
+        status: {
+            schemaDoc: 'Sending status: pending (waiting to send/retry), success (sent successfully), failed (permanently failed after TTL expired)',
+            type: 'Select',
+            dataType: 'string',
+            options: WEBHOOK_PAYLOAD_STATUSES,
+            defaultValue: WEBHOOK_PAYLOAD_STATUS_PENDING,
+            isRequired: true,
+        },
+
+        attempt: {
+            schemaDoc: 'Number of send attempts made (starts at 0)',
+            type: 'Integer',
+            defaultValue: 0,
+            isRequired: true,
+        },
+
+        lastHttpStatusCode: {
+            schemaDoc: 'HTTP status code from the last send attempt',
+            type: 'Integer',
+            isRequired: false,
+        },
+
+        lastResponseBody: {
+            schemaDoc: `Response body from the last send attempt (truncated to ${WEBHOOK_PAYLOAD_MAX_RESPONSE_LENGTH} chars)`,
+            type: 'Text',
+            isRequired: false,
+        },
+
+        lastErrorMessage: {
+            schemaDoc: 'Error message from the last send attempt',
+            type: 'Text',
+            isRequired: false,
+        },
+
+        expiresAt: {
+            schemaDoc: 'Timestamp after which no more send attempts will be made',
+            type: 'DateTimeUtc',
+            isRequired: true,
+            hooks: {
+                resolveInput: ({ resolvedData, operation, fieldPath }) => {
+                    if (operation === 'create' && !resolvedData[fieldPath]) {
+                        return dayjs().add(WEBHOOK_PAYLOAD_TTL_DAYS, 'day').toISOString()
```

Comment:
Can we compute this as `createdAt + WEBHOOK_PAYLOAD_TTL_DAYS`?

Reply:
Looks like it doesn't need to be changed. Using dayjs() during creation is functionally equivalent to createdAt + TTL since they happen at the same moment. The worst thing that can happen is that expiresAt may be a liiiiiitle more than createdAt (microseconds).